hats/details.md

```diff
@@ -2,32 +2,52 @@
-  * « In the future, for performance reasons, do you wish me setup Firefox to pause all other webpages while you're playing? Here's what you can gain.»;
-  * « In the future, for privacy reasons, do you wish me to setup Firefox to prevent Facebook from tracking you around the web using your connection data? Here's what you can gain. »;
-  * « In the future, for privacy reasons, do you wish me to automatically turn on Privacy mode when you are browsing this dating site? »
+3. Once Firefox has learnt a bit about the user's behavior, a Suggested Tile offers to setup Firefox automatically:
+  * « Firefox for Banking, by Mozilla »
+    * Details: « You use Firefox to access your banking site. Firefox can configure itself to improve your security when you are browsing this site. This is entirely automatic and this does not affect your experience with other websites. »
+  * « Firefox for Facebook, by Mozilla »
+    * Details: « You use Facebook. Firefox can configure itself to improve your experience with Facebook, while better protecting your privacy. This is entirely automatic and this does not affect your experience with other websites. »
+  * « Firefox for Gaming, by Mozilla »
+    * Details: « You use Firefox for gaming. Firefox can configure itself to provide the best gaming experience. This is entirely automatic and this does not affect your experience with other websites. »
+  * « Firefox for Privacy, by Mozilla »
+    * Details: « You use Firefox for browsing a dating site. Firefox can configure itself to improve your privacy when you are browsing this site. This is entirely automatic and this does not affect your experience with other websites. »
   * ...
-4. At this stage, we have a multi-headed Firefox, each head wearing a different Hat. Firefox learns how websites need to be opened with specific Hats;
-5. Each Hat behaves as if it were upgraded semi-independently, with new features/settings that match specific needs:
+4. If a user accepts the suggestion of the Suggested Tile, this Tile becomes a Permanent Tile;
+5. Clicking on either the Suggested Tile or the Permanent Tile switches to a *Firefox Hat*;
+6. A *Firefox Hat* is basically an instance of Firefox window with its own visual theme and configured specifically for one website or a collection of websites;
+7. Each Hat behaves as if it were upgraded semi-independently, with new features/settings that match specific needs:
   * As Firefox gains new WebVR features, Gaming-oriented Hats receive a Suggested Tile that offers users the ability to turn on experimental features;
-  * As Firefox gains new privacy protection features, Hats that already have some Privacy settings on turn have these features turned on automatically;
-  * Some new privacy protection features are developped as add-ons, which appear as Suggested Tiles for Privacy-aware hats;
+  * As Firefox gains new privacy protection features, Privacy-oriented Hats turn have these features turned on automatically;
```

Comment:
I think this should answer: why don't we turn on the feature for everyone right away?

I've been phrasing it like "*aggressive* privacy protection features" to emphasize that these are features that may go beyond what we can "safely" do (e.g., they might break sites, endanger ad revenue, etc).  Another angle: we can be more vocal about the features.  Informing users about the protections we've added, tracking we have avoided, etc., will be specifically welcomed by Firefox for Privacy users who were looking for exactly those features.